docs/faq.md

```diff
@@ -66,4 +60,45 @@ TBD
 ### Is Luigi already being used within any products, or is it still too new?
 
 Yes, it is already being used in production and close-to-production within SAP. For example in Kyma, SAP C/4HANA Cockpit, Context Driven Services, Konduit and Varkes. Outside of SAP, SAAS AG (partner) uses Luigi. Additionally, there are some POCs going on and we're supporting a few other customers and partners who want to start using Luigi soon.
+
+
+### Can I authenticate Luigi with Google Cloud Identity?
+
+Yes, to use Luigi with a google account, follow these steps:
+
+1. Register a project and generate an OAuth2 Web Client based on [Google Developers Identity - OAuth2UserAgent](https://developers.google.com/identity/protocols/OAuth2UserAgent).
+2. To get your app running locally, set the Autorized JavaScript Origins URIs to `http://localhost:4200` and Authorized redirect URIs to `http://localhost:4200/luigi-core/auth/oauth2/callback.html?storageType=localStorage`.
+3. Copy the Client ID on the right side, ending with `apps.googleusercontent.com`
+4. Update the LuigiConfig auth section. We have added also the parts for logout and getting user information.
+   ```javascript
+  {
+    auth: {
+      use: 'oAuth2ImplicitGrant',
+      oAuth2ImplicitGrant: {
+        authorizeUrl: 'https://accounts.google.com/o/oauth2/v2/auth',
+        oAuthData: {
+          client_id: 'YOUR_CLIENT_ID...apps.googleusercontent.com',
+          scope: 'openid https://www.googleapis.com/auth/userinfo.email profile',
+        }
+      },
+      userInfoFn: async (settings, authData) => {
```

Comment:
I think it's not needed, as basic identity data are returned in `id_token`. Having it here adds unneeded complexity

Reply:
It is required to show the picture of the user, so maybe I put it in a separate code block below.